### DISCUSSION TITLE
Unexpected crashes every 20 minutes, how can I investigate?

### DISCUSSION THREAD

**Discussion by Callumk7** [neovim/neovim#24380] [Q&A] [answered]
Hi all, 

Neovim over the last couple of days has been unexpectantly crashing (as in the process ends and I am returned to the commandline) without warning, about once every 20-30 minutes. 

I appreciate this is really unhelpful, how can I investigate the root cause further? Are there logs that might help me shed light on what is causing this? 

I've had no problems like this before and I am struggling to think of any changes that I have made to my config recently, so any help in how to diagnose the problem would be greatly appreciated.

**Comment by clason** [comment, marked as answer]
First step is always to 
1. minimize your config (either disable stuff, or -- better -- use `nvim --clean -u minimal.lua`)
2. try to get exact reproducing steps
3. try to get a stacktrace

See https://github.com/neovim/neovim/blob/master/CONTRIBUTING.md#reporting-problems
